### PR TITLE
fix(charts,table): replace or use exact lodash imports

### LINKED ISSUE #4750: Remove lodash from patternfly
Recently we did an analysis on how many functions are used by patternfly from lodash.  There is less than a handful.  The size of lodash is larger then our entire component library if you do a bundle analysis.  We should remove this dependence to improve load times, reduce our foot print, and not increase consumers bundle size by including this library.

### PR DESCRIPTION
**Pull request by redallen**
**What**: Closes #4750. Reduces bundle sizes up to 500kb.

**Additional issues**:

### CODE REVIEW

File: packages/react-charts/src/components/ChartCursorTooltip/ChartCursorFlyout.tsx [hunk 1]
```diff
@@ -1,7 +1,8 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { Helpers, CommonProps, Path } from 'victory-core';
-import { isPlainObject, assign } from 'lodash';
+import isPlainObject from 'lodash/isPlainObject';
+import assign from 'lodash/assign';
```
Comment (KKoukiou) [inline packages/react-charts/src/components/ChartCursorTooltip/ChartCursorFlyout.tsx:5]: You can remove this and use Object.assign instead.

**Review by KKoukiou — DISMISSED** [review]
Nice job!

**Review by ddonahue007 — DISMISSED** [review]
LGTM

**Review by mturley — APPROVED** [review]
LGTM! Nice!

File: packages/react-charts/src/components/ChartUtils/chart-padding.ts [hunk 2]
```diff
@@ -1,17 +1,14 @@
...
-  if (!isEmpty(padding)) {
-    return get(padding, side, 0);
-  }
-
-  if (isFinite(padding)) {
+  if (typeof padding == 'number') {
     return padding as number;
+  } else if (typeof padding == 'object') {
+    return padding[side] || 0;
   }
```
Comment (dlabrecq) [inline packages/react-charts/src/components/ChartUtils/chart-padding.ts:12]: I'm not sure this change is really the equivalent of what we had in place? We want to ensure padding is truly empty, not just an object, and there may be one or more properties.

**Review by dlabrecq — CHANGES_REQUESTED** [review]

File: packages/react-charts/src/components/ChartUtils/chart-padding.ts (hunk 2 above)
Comment (redallen) [reply packages/react-charts/src/components/ChartUtils/chart-padding.ts]: Fixed!

**Review by dlabrecq — COMMENTED** [review]
LGTM

**Review by dlabrecq — APPROVED** [review]

**Review by mturley — APPROVED** [review]

**Review by ddonahue007 — COMMENTED** [review]
LGTM

**Pull request merged**